### PR TITLE
Add TypeScript definition

### PR DESCRIPTION
**Pull request by BendingBender**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,10 +13,11 @@
 		"node": ">=6"
 	},
 	"scripts": {
-		"test": "xo && ava"
+		"test": "xo && ava && tsd-check"
```
Comment (Qix-) [inline package.json]: Might be nice to run `tsd-check` before the tests. Just a suggestion.

Comment (BendingBender) [reply package.json]: Hm, I don't have a preference about this. @sindresorhus what do you think?

**Comment by BendingBender** [comment]
> Is this one better?

This is better because:

- the types here will be updated along with the interface changes (the definitions on DT are already out of date)
- breaking changes on DT might happen even in patch releases, introducing a breaking chage here will necessitate a major release

**Comment by Qix-** [comment]
I don't doubt the reasons for including here, I'm merely asking if the type definitions are complete or better than those on DT.

**Comment by BendingBender** [comment]
> I don't doubt the reasons for including here, I'm merely asking if the type definitions are complete or better than those on DT.

The definitions on DT are out of date. The ones I've added are complete and up to date.

File: package.json (hunk 1 above)
Comment (sindresorhus) [reply package.json]: Running `tsd-check` is the least important of the 3 binaries for a JS project, and should be last like it is now.

**Pull request merged**